Modules/_winapi.c

```diff
@@ -464,6 +474,41 @@ _winapi_CreateFile_impl(PyObject *module, LPCTSTR file_name,
     return handle;
 }
 
+/*[clinic input]
+_winapi.CreateFileMappingW -> HANDLE
```

Comment:
Personally, I'm not against using the real name with the "W" suffix. But thus far this module has used the macro names such as `_winapi.CreateFileMapping`.

Actually, we don't define `UNICODE`, and we have a few functions in _winapi that link to the ANSI version and use `LPCTSTR` -> `LPCSTR` (i.e. `const char *`). This includes `_winapi.CreateFile`. It should instead call `CreateFileW` and use an `LPCWSTR` name.

Reply:
Thanks for making a point of this.  I would prefer to stick with the established naming pattern unless there was a real reason otherwise.  I have changed it to ``_winapi.CreateFileMapping`` as you suggest and likewise dropped the "W" on ``_winapi.OpenFileMapping`` in commit 68785333a25d902a42252ace3a8268357efd6094.